src/wirecloud/platform/static/js/wirecloud/wiring/Operator.js

```diff
@@ -116,7 +116,10 @@
             build_endpoints.call(this);
             build_prefs.call(this, businessInfo.preferences);
 
-            this.logManager.log(utils.gettext("The operator was created successfully."), Wirecloud.constants.LOGGING.INFO_MSG);
+            this.logManager.log(utils.gettext("The operator was created successfully."), {
+                level: Wirecloud.constants.LOGGING.INFO_MSG,
+                console: false
```

Comment:
Yes, it does. Check it out!
https://github.com/Wirecloud/wirecloud/blob/develop/src/wirecloud/platform/static/js/wirecloud/LogManager.js#L80